lib/instruments.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
         const { input, instrument = 'acoustic_grand_piano' } = chunk
 
         function pathToAudio(note) {
-            return path.join(pathToSoundfonts, `${instrument}-mp3`, `${note.toString().toUpperCase()}.mp3`)
+            const noteString = note.toString();
+            return path.join(pathToSoundfonts, `${instrument}-mp3`, `${noteString.charAt(0).toUpperCase() + noteString.slice(1)}.mp3`)
```

Comment:
Actually, that won't work for `play(eB)`. Think we need to `toString().toLowerCase().replace(...)`